src/modules/reporting/actions/submit-initial-report.js

```diff
@@ -1,9 +1,10 @@
 import { augur } from 'services/augurjs'
-import noop from 'utils/noop'
+import { REPORTING_REPORTING } from 'modules/routes/constants/views'
+import makePath from 'modules/routes/helpers/make-path'
 import logError from 'utils/log-error'
 import { getPayoutNumerators } from 'modules/reporting/selectors/get-payout-numerators'
 
-export const submitInitialReport = (marketId, selectedOutcome, invalid, callback = logError) => (dispatch, getState) => {
+export const submitInitialReport = (marketId, selectedOutcome, invalid, history, callback = logError) => (dispatch, getState) => {
```

Comment:
Redirect worked! Should the market still appear in the designated report list?

<img width="1655" alt="screen shot 2018-02-27 at 1 04 38 pm" src="https://user-images.githubusercontent.com/328965/36754761-d2bf11f2-1bbe-11e8-8b2d-58821a6d0e24.png">

Reply:
Right! Need to refrsh market status. Or should it happen automagically! Will reseach

Reply:
@bthaile Keep it here or open another task?

Reply:
I’ll open another task

Reply:
Yea, issue might be with augur-node or UI isn’t handling the re-emit log correctly.

Reply:
Sweet. Thanks!